addons/common/fnc_getUniquePlayerItems.sqf

```diff
@@ -0,0 +1,44 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getUniquePlayerItems
+
+Description:
+    A function used to retrieve a unique list of items in the units inventory
```

Comment:
~A function used to~ [R]etrieve[s] a unique list of items in the units inventory[.]